webviz_subsurface/plugins/_tornado_plotter_fmu/shared_settings/_filters.py

```diff
@@ -0,0 +1,156 @@
+                    clearable=False,
+                )
+            )
+
+        # Creates dropdowns for any data columns added as single value filters
+        for selector in self._multi_filters:
+            values = self._table_provider.get_column_data([selector])[selector].unique()
+            elements.append(
+                wcc.Dropdown(
+                    label=selector,
+                    id={
+                        "id": self.multi_filter_id,
+                        "name": selector,
+                        "type": "multi_filter",
+                    },
+                    options=[{"label": val, "value": val} for val in values],
+                    value=values,
+                    multi=True,
+                )
+            )
+        return elements
+
+    def set_callbacks(self) -> None:
+        @callback(
+            Output(
+                self.get_store_unique_id(PlugInIDs.Stores.DataStores.TORNADO_DATA),
+                "data",
+            ),
+            Input(
+                self.get_store_unique_id(PlugInIDs.Stores.Selectors.RESPONSE),
+                "data",
+            ),
+            Input(
+                {
+                    "id": self.single_filter_id,
+                    "name": ALL,
+                    "type": "single_filter",
+                },
+                "value",
+            ),
+            Input(
+                {
+                    "id": self.multi_filter_id,
+                    "name": ALL,
+                    "type": "multi_filter",
+                },
+                "value",
+            ),
+        )
+        def _update_tornado_with_response_values(
+            response: str, single_filters: List, multi_filters: List
+        ) -> str:
+            """Returns a json dump for the tornado plot with the response values per realization"""
+
+            data = self._table_provider.get_column_data(
+                [response] + self._single_filters + self._multi_filters
+            )
+
+            # Filter data
+            if single_filters is not None:
+                for value, input_dict in zip(
+                    single_filters, callback_context.inputs_list[1]
+                ):
+                    data = data.loc[data[input_dict["id"]["name"]] == value]
+            if multi_filters is not None:
+                for value, input_dict in zip(
+                    multi_filters, callback_context.inputs_list[2]
+                ):
+                    data = data.loc[data[input_dict["id"]["name"]].isin(value)]
+
+            return json.dumps(
+                {
+                    "ENSEMBLE": self._ensemble_name,
+                    "data": data.groupby("REAL")
+                    .sum()
+                    .reset_index()[["REAL", response]]
+                    .values.tolist(),
+                    "number_format": "#.4g",
+                }
+            )
+
+    def uuid(self, element: Optional[str] = None) -> str:
```

Comment:
Where is this method used? I could not find any reference to it.

Reply:
It is not used anymore, just forgot to remove it:)